_posts/2017-01-06-conduit-0.2.1.md

```diff
@@ -3,14 +3,12 @@ title: "Conduit 0.2.1 Released"
 tags: new-release
 ---
 
-Version 0.2.1 of Conduit is now available at: https://software.llnl.gov/conduit/releases.html
-
 Conduit provides APIs focused on simplifying data exchange in HPC simulations. It provides an intuitive model for describing hierarchical scientific data in C++, C, Fortran, and Python and is used for data coupling between packages in-core, serialization, and I/O tasks.
 
-The 0.2.1 version is primarily a bug fix releases, which includes:
+The [0.2.1 version](https://llnl-conduit.readthedocs.io/en/latest/) is primarily a bug fix releases, which includes:
 
 - Fixes to support static builds, including on BGQ using xlc and gcc
 - Bug fixes for Blueprint verify routines.
 - The elimination of separate Fortran libs by moving Fortran symbols into their associated main libs
 
-For more details, see Conduit's [Release Notes](https://software.llnl.gov/conduit/releases.html#v0-2-1).
+For more details, see Conduit's [Release Notes](https://github.com/LLNL/conduit/releases).
```

Comment:
The same goes for this link, since the post as a whole is version specific.
https://github.com/LLNL/conduit/releases/tag/v0.2.1

Reply:
Agreed.